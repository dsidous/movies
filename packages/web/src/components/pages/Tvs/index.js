import { compose, withProps } from 'recompose';
import { Shows } from '@movies/ui';
import { withTvGenres } from '@movies/common';

export default compose(
  withProps(() => ({ media: 'tvs', resultMedia: 'tv' })),
  withTvGenres(),
)(Shows);
