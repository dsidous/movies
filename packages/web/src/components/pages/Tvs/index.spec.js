import React from 'react';
import { shallow } from 'enzyme';
import Tvs from '.';

const defaultProps = {
  location: { search: '1' },
};

describe('Pages/Tvs', () => {
  it('should render as expected', () => {
    const wrapper = shallow(<Tvs {...defaultProps} />);

    expect(wrapper).toMatchSnapshot();
  });
});
