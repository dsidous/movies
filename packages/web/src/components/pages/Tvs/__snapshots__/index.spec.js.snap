// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/Tvs should render as expected 1`] = `ShallowWrapper {}`;
